packages/hash/api/src/graphql/typeDefs/blockprotocol.typedef.ts

```diff
@@ -0,0 +1,48 @@
+import { gql } from "apollo-server-express";
+
+export const blockprotocolTypedef = gql`
+  type BlockType {
+    entryPoint: String!
+    tagName: String
+  }
+
+  type BlockVariant {
+    description: String
+    icon: String
+    name: String
+    properties: JSONObject
+    examples: [JSONObject]
```

Comment:
Examples should never be `[null]`
```suggestion
    examples: [JSONObject!]
```